utils/python/CIME/check_input_data.py

```diff
@@ -186,10 +186,9 @@ def check_input_data(case, svn_loc=None, input_data_root=None, data_list_dir="Bu
                             else:
                                 no_files_missing = False
                         else:
-                            logging.info("  Already had input file: '%s'" % full_path)
+                            logging.debug("  Already had input file: '%s'" % full_path)
 
                 else:
                     model = os.path.basename(data_list_file).split('.')[0]
                     logging.warning("Model %s no file specified for %s"%(model,description))
-
```

Comment:
It's nice to have a whitespace at the end of python blocks since python lacks the standard {} syntax.